lib/nerves_hub_cli/api.ex

```diff
@@ -5,6 +5,11 @@ defmodule NervesHubCLI.API do
   @file_chunk 4096
   @progress_steps 50
 
+  @castore_certs CAStore.file_path()
+                 |> File.read!()
+                 |> X509.from_pem()
+                 |> Enum.map(&X509.Certificate.to_der/1)
+
```

Comment:
@fhunleth @jjcarstens do either of you have any concerns with this?

Reply:
It's ok. This is a really big PR that looks like a clear improvement so I mostly want to get it in and iterate.